pkg/syncer/util.go

```diff
@@ -127,6 +127,7 @@ func IsValidVolume(ctx context.Context, volume v1.Volume, pod *v1.Pod, metadataS
 // if volumeIds is empty, then all volumes from CNS will be retrieved by pagination
 func getQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clusterID string, volumeManager volumes.Manager) ([]*cnstypes.CnsQueryResult, error) {
 	log := logger.GetLogger(ctx)
+	log.Debugf("FullSync: getQueryResults is called with volumeIds %v for clusterID %s", volumeIds, clusterID)
```

Comment:
Only if this is used in the full sync it makes sense to prefix the message with `FullSync:`.
If yes keep this as it and address refactoring of the function name.

Reply:
Done.